clang/lib/CConv/DeclRewriter.cpp

```diff
@@ -108,52 +116,53 @@ void DeclRewriter::rewriteDecls(ASTContext &Context, ProgramInfo &Info,
   // Do the declaration rewriting
   DeclRewriter DeclR(R, Context, GVG);
   DeclR.rewrite(RewriteThese, TouchedFiles);
+
+  for (const auto *R : RewriteThese)
+    delete R;
```

Comment:
How confident are you that this is the right thing to do? :)

Reply:
Fairly confident. I had to `new` all the replacements, so I've got to `delete` them here to avoid leaks. The replacements definitely will not be used after this method returns.